tensorflow_encrypted/protocol/pond.py

```diff
@@ -45,20 +47,42 @@
 _thismodule = sys.modules[__name__]
 
 
+def import_native() -> None:
```

Comment:
i'm in favour of doing this the right way now tbh since we're soon going to be offering several backing tensors. jump on a call today when you guys are up?

Reply:
Yep, have an idea on how we could do this the right way myself. Best to chat and pick the right one!